project/Doc.scala

```diff
@@ -75,7 +78,7 @@ object UnidocRoot extends AutoPlugin {
       |  }
       |</script>""".stripMargin.replaceAll("\n", "\\\\n").replaceAll("\"", "\\\\\"")
 
-  override lazy val projectSettings = javaUnidocSettings ++ Seq(
+  override lazy val projectSettings = scalaJavaUnidocSettings ++ Seq(
     unidocAllSources in (JavaUnidoc, unidoc) ++= allGenjavadocSources.value,
     unidocAllSources in (JavaUnidoc, unidoc) := {
```

Comment:
We're excluding all of the `internal` packages here in Javadoc, but not for Scala.